.github/workflows/rule-validate.yml

```diff
@@ -314,12 +340,12 @@ jobs:
             -d "$body"
 
       - name: Wait for MQL Mimic check to be completed
-        uses: fountainhead/action-wait-for-check@v1.1.0
-        id: wait-for-build
+        uses: sublime-security/action-wait-for-check@master
```

Comment:
I forked this to add support for `ignoreIDs`. This is pretty simple so I'm hopeful it'll work.

The issue I saw was that we triggered the new MQL Mimic run, but immediately saw the previous result and finished the workflow (it needs to stay running for MQL Mimic to post back results).

Ignoring the existing check ID should avoid this issue. I validated that new checks show up with a new ID, even when the same name is posted to the same ref/commit.

I'd probably do this differently if I was starting from scratch -- have the pending check be created here (vs. from MQL mimic) and pass along the check ID to be updated, but that would require juggling changes in a couple spots at once.